docs/_content/themas/achtergronddocumentatie/versies-en-migraties.md

```diff
@@ -44,25 +48,24 @@ HTTP 200
 }
 ```
 
-1. Huidige situatie: Zaken API v1.0.0
-2. Een Zaak heeft een Status (beide in de Zaken API)
-3. Zaken API v2.0.0 komt uit
+3. De provider heeft er voor gekozen de URI's naar gerelateerde objecten in een koppeltabel in de database vast te leggen.
+4. De provider gaat de Zaken API v2.0.0 ondersteunen en zal op termijn gaan stoppen met de ondersteuning van v1.0.0.
 
 **Wat gaat er gebeuren?**
 
-1. Zaken API v1.0.0 zal nog even moeten bestaan naast v2.0.0
-2. Verzoeken op de Zaken API v1.0.0 geven een waarschuwing mee als header
-   dat er een nieuwe versie is.
-3. Zaak met UUID `6c821f` moet beschikbaar zijn op de Zaken API v1.0.0 en 
-   v2.0.0
-4. Verwijzingen binnen dezelfde API wijzen naar dezelfde versie. Ofwel, een 
+1. De provider zal de Zaken API v1.0.0 nog enige tijd naast v2.0.0 moeten ondersteunen.
+2. Vanaf het beschikbaar komen van de Zaken API v2.0.0 geven verzoeken op de Zaken API v1.0.0 een waarschuwing mee als header
+   waarin staat dat er een nieuwe versie beschikbaar is.
+3. In de koppeltabel is voor het object met de UUID's '8aa57d' vastgelegd welke v1.0.0 en welke v2.0.0 URI van toepassing is. 
```

Comment:
Deze UUIDs komen uit het niets. Wellicht weglaten of "bijvoorbeeld" toevoegen.

Mijn originele UUID kwam uit de GET bij de situatie schets.

Reply:
Je hebt helemaal gelijk. Ga ik iets aan doen. Ik begrijp zelf ook niet waarom ik hier de UUID heb aangepast.

Reply:
Even iets te snel geantwoord. Het UUID '8aa57d' komt toch niet uit de lucht vallen. Het staat immers ook hierboven in je eigen voorbeeldbericht dat niet is aangepast door mij.